lib/boom_notifier.ex

```diff
@@ -65,40 +67,46 @@ defmodule BoomNotifier do
       end
 
       # Notifiers validation
-      walkthrough_notifiers(
-        unquote(config),
-        fn notifier, options -> validate_notifiers(notifier, options) end
-      )
+      walkthrough_notifiers(fn notifier, options -> validate_notifiers(notifier, options) end)
 
       def notify_error(conn, %{kind: :error, reason: %mod{}} = error) do
-        settings = unquote(config)
-        {ignored_exceptions, _settings} = Keyword.pop(settings, :ignore_exceptions, [])
-
-        unless Enum.member?(ignored_exceptions, mod) do
-          do_notify_error(conn, settings, error)
+        unless Enum.member?(Config.ignore_exceptions(), mod) do
+          do_notify_error(conn, error)
         end
       end
 
       def notify_error(conn, error) do
-        do_notify_error(conn, unquote(config), error)
+        do_notify_error(conn, error)
+      end
+
+      def manual_notify_error(conn, error) do
```

Comment:
What about using the already existing `notify_error` function? I think all the logic there applies here (limits & filters for exceptions). So, we would have similar ways in router and controller to notify errors when exceptions are being intercepted by user code (`error_handler` or simple `try-catch-rescue` blocks).